pom.xml

```diff
@@ -56,6 +56,21 @@
             <artifactId>junit-jupiter</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>4.5.14</version>
```

Comment:
**Suggestion**: Would be recommended to move the hardcoded dependency versions (e.g., httpclient version 4.5.14) to a <properties> block.
**Reason**:
This centralises version management, making it easier to update dependencies across the project in the future. Here's an example:
```
<properties>
    <httpclient.version>4.5.14</httpclient.version>
</properties>
```